uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1540,6 +1540,42 @@ sub my_trim {
 	return $str;
 }
 
+=pod
+
+=head3 fetch_minc_header_info($minc, $field, $awk, $keep_semicolon)
+
+Function that fetches header information in MINC file.
+
+INPUTS:
+  - $minc : MINC file
+  - $field: field name to look for in MINC header (or 'all' to grep all headers)
+  - $awk  : awk info to check if argument inserted in MINC header
+  - $keep_semicolon: if set, keep ";" at the end of extracted value
+
+RETURNS: value of the field found in the MINC header
+
+=cut
+
+sub fetch_header_info {
+    my ($minc, $field, $awk, $keep_semicolon) = @_;
+    my $value;
+    if ($field eq 'all') {
+        # run mincheader and return all the content of the command
+        $value = `mincheader -data "$minc"`;
+    }
+    else {
+        # fetch a particular header value, remove extra spaces and optionally
+        # the semicolon
+        my $val = `mincheader $minc | grep $field | awk '{print $awk}' | tr '\n' ' '`;
```

Comment:
Since we know that MINC headers (and correct me if I'm wrong) are of the form

```perl
        xxxxxx:yyyyyy = value ;
```

where value can be surrounded by quotes, we can use this command instead

```
my $val = `mincheader $minc | grep $field | cut -d= -f2 | tr '\n' ' '`;
```

This will enable us to get rid of the `$awk` parameter.

Reply:
done